zkevm-circuits/src/evm_circuit/execution/return.rs

```diff
@@ -0,0 +1,48 @@
+use crate::{
+    evm_circuit::{
+        execution::ExecutionGadget,
+        step::ExecutionState,
+        util::{constraint_builder::ConstraintBuilder, CachedRegion, Cell},
+        witness::{Block, Call, ExecStep, Transaction},
+    },
+    util::Expr,
+};
+use eth_types::Field;
+use halo2_proofs::plonk::Error;
+
+#[derive(Clone, Debug)]
+pub(crate) struct ReturnGadget<F> {
+    opcode: Cell<F>,
+}
+
+impl<F: Field> ExecutionGadget<F> for ReturnGadget<F> {
+    const NAME: &'static str = "RETURN";
+
+    const EXECUTION_STATE: ExecutionState = ExecutionState::RETURN;
+
+    fn configure(cb: &mut ConstraintBuilder<F>) -> Self {
+        let opcode = cb.query_cell();
+        cb.opcode_lookup(opcode.expr(), 1.expr());
+
+        // Other constraints are ignored now for RETURN to serve as a mocking
```

Comment:
```suggestion
        // TODO: Other constraints are ignored now for RETURN to serve as a mocking
```
I think having the keyword `TODO` helps finding these comments :)

Reply:
Added in [`f77e67d`](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/419/commits/f77e67d0e8f3d30d401b2a66a8d8a915e6348156).